Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM mjstealey/hydroshare_base
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
-
+RUN pip uninstall -y django-celery

```

Comment:
if you don't need to use this, it should be removed from the requirements not uninstalled afterwards.